### PR TITLE
First attempt at an Essential Skills guide

### PR DESCRIPTION
**Pull request by clairbarrass**
I've recreated the Quick Ref Sheet, without the PBS section.
May be good to add a Slurm bit at the end.
Am uploading the source .docx to make maintenance easier but only linked to PDF version.

### CODE REVIEW

**Comment by clairbarrass** [comment]
p.s. if someone wants to give me text for the Slurm bit, I am happy to add it in an format it.

File: README.md [hunk 1]
```diff
@@ -1,8 +1,6 @@
 # ARCHER2 Documentation
 
-\*\*Note: The ARCHER2 Service is not yet available. This documentation
-is in development. ARCHER2 is due to commence operation in 2020,
-replacing the current service ARCHER. For more information on ARCHER,
+\*\* This documentation is in development. ARCHER2 is currently running as a small early access system while the full system is being commissioned. For more information on ARCHER,
 please visit the [ARCHER web site](http://www.archer.ac.uk).\*\*
```
Comment (jsindt) [inline README.md:4]: Should we point them to ARCHER2 website instead of ARCHER>

File: docs/user-guide/connecting.md [hunk 2]
```diff
@@ -34,11 +34,15 @@ some Windows OS. Also make sure to check whether Windows Firewall hasn't blocked
...
+    - If you are using a "managed desktop" machine, so do not have admin rights, you can use the [Portable edition of MobaXterm](https://mobaxterm.mobatek.net/download-home-edition.html) which doesn't need install privilages.
+
+    - If this is your first time using MobaXterm, check that a permanent /home directory has been set up (or all saved info will be lost from session to session). Go to "Settings" -> "Configuration"-> check path to "Persistent home directory" is set and make sure path is "private" if prompted.
+    - Any ssh key generated in MobaXterm will, by default, be stored in the permananet /home directory (see above) i.e. if your /home directory is `_MyDocuments_\MobaXterm\home` then within that folder you will find `_MyDocuments_\MobaXterm\home\.ssh` with your keys.  This folder will be 'hidden' by default so you may need to tick 'Hidden items' under 'View' in Windows Explorer to see it.
```
Comment (jsindt) [inline docs/user-guide/connecting.md:43]: Typo: "pemananet" should be "permanent"

File: docs/essentials/index.md [hunk 3]
```diff
@@ -1,12 +1,45 @@
...
+
+## Text Editors
+
+The following text editors are available on ARCHER2
+
+| Name | Description | Examples |
+| ---  | ---         | ---      |
+| emacs | A widely used editor <br /> with a focus on extensibility. | emacs    -nw    sharpen.pbs <br /> CTRL+X CTRL+C quits  <br /> CTRL+X CTRL+S saves
+| nano | A small, free editor  <br />with a focus on user friendliness.|nano sharpen.pbs <br /> CTRL+X quits <br /> CTRL+O saves|
+| vi | A mode based editor <br /> with a focus on aiding code development. | vi cfd.f90  <br /> :q in command mode quits  <br /> :q! in command mode quits without saving  <br />:w in command mode saves  <br /> i in command mode switches to insert mode  <br /> ESC in insert mode switches to command mode |
```
Comment (jsindt) [inline docs/essentials/index.md:34]: Maybe worth using backquotes for commands for text editors? That way commands stand out a bit better.

**Review by jsindt — CHANGES_REQUESTED** [review]
Overall looks good. I've suggested some minor changes:
1) Line 48 of `docs/user-guide/connecting.md` has typo
2) `docs/essentials/index.md`: it might look better to use backquotes for the Text Editor instructions (e.g. `:w` instead of :w for vi) so that instructions pop more
3) README.md: links to ARCHER -- do we still need this?

I'm happy to be overruled on (2) and (3)

File: README.md (hunk 1 above)
Comment (clairbarrass) [reply README.md:4]: Good shout

File: docs/user-guide/connecting.md (hunk 2 above)
Comment (clairbarrass) [reply docs/user-guide/connecting.md:43]: Doh

File: docs/essentials/index.md (hunk 3 above)
Comment (clairbarrass) [reply docs/essentials/index.md:34]: Good idea

**Comment by clairbarrass** [comment]
All good spots - I believe I have remedied all.

**Review by jsindt — COMMENTED** [review]
LGTM

**Review by jsindt — APPROVED** [review]

**Pull request merged**